builds/e2e/proxy/proxy-deployment-template.json

```diff
@@ -26,19 +26,19 @@
 			"type": "bool"
 		},
 		"linux_vm_creation_template_uri": {
-			"defaultValue": "https://raw.githubusercontent.com/Azure/iotedge/master/builds/e2e/proxy/create-linux-vm-template.json",
+			"defaultValue": "https://raw.githubusercontent.com/Azure/iotedge/main/builds/e2e/proxy/create-linux-vm-template.json",
 			"type": "string"
 		},
 		"proxy_config_script_uri": {
-			"defaultValue": "https://raw.githubusercontent.com/Azure/iotedge/master/builds/e2e/proxy/configure_proxy.sh",
+			"defaultValue": "https://raw.githubusercontent.com/Azure/iotedge/main/builds/e2e/proxy/configure_proxy.sh",
 			"type": "string"
 		},
 		"runner_config_script_uri": {
-			"defaultValue": "https://raw.githubusercontent.com/Azure/iotedge/master/builds/e2e/proxy/configure_runner.sh",
+			"defaultValue": "https://raw.githubusercontent.com/Azure/iotedge/main/builds/e2e/proxy/configure_runner.sh",
 			"type": "string"
 		},
 		"create_ssh_keys_script_uri": {
-			"defaultValue": "https://raw.githubusercontent.com/Azure/iotedge/master/builds/e2e/proxy/create_ssh_keys.sh",
+			"defaultValue": "https://raw.githubusercontent.com/Azure/iotedge/main/builds/e2e/proxy/create_ssh_keys.sh",
```

Comment:
Whys is 1.2 refering to scripts from `main` branch?

Reply:
Because 1.2 forked from master. We could do a search & replace whenever we fork a new branch, but in many cases it's not worth the effort. In this case we don't expect the ARM template files to be different between branches (these instructions don't really depend on changes in our code), so it's fine to refer to the default branch.